store/store.go

```diff
@@ -1059,9 +1060,9 @@ func (s *Store) ListRefresh(installed []*RefreshCandidate, user *auth.UserState)
 		Data:        jsonData,
 	}
 
-	if os.Getenv("SNAPPY_USE_DELTAS") == "1" {
+	if os.Getenv("SNAPPY_USE_DELTAS_EXPERIMENTAL") == "1" {
```

Comment:
can you make it SNAPD_ while you're at it? sorry to be nitpicky, and I know there are some env vars that are SNAPPY_, but I'm trying to have things that only affect snapd be SNAPD_